lib/menus/utils/image-util.js

```diff
@@ -0,0 +1,102 @@
+import { layouts } from 'minecraft-inventory-gui/lib/layouts.mjs'
+
+const invspritePositions = require('../../invsprite.json')
+
+globalThis.layouts = layouts
+
+if (window.location.href.includes('127.0.0.1')) {
+  var IMAGE_ROOT = 'textures/'
+} else {
+  var IMAGE_ROOT = 'https://raw.githubusercontent.com/PrismarineJS/minecraft-assets/master/data/1.16.4/'
+}
+
+function patchPath(path) {
+  if (IMAGE_ROOT.includes('.com')) {path = path.replace('block/', 'blocks/'); path=path.replace('item/', 'items/')}
+  //if (path.includes('enchant_table_anim'))path='enchant_table_anims2'
+  console.log(path)
+  return path
+}
+
+window.loadedImageBlobs = {}
+
+const images = [
+  'item/brick',
```

Comment:
Why is this hard coded?

Reply:
That was left over from when those items were the only ones supported by the library.